internal/rootcoord/create_collection_task.go

```diff
@@ -101,8 +101,13 @@ func (t *createCollectionTask) validateSchema(schema *schemapb.CollectionSchema)
 	if t.Req.GetCollectionName() != schema.GetName() {
 		return fmt.Errorf("collection name = %s, schema.Name=%s", t.Req.GetCollectionName(), schema.Name)
 	}
-	if hasSystemFields(schema, []string{RowIDFieldName, TimeStampFieldName}) {
-		return fmt.Errorf("schema contains system field: %s, %s", RowIDFieldName, TimeStampFieldName)
+	if hasSystemFields(schema, []string{RowIDFieldName, TimeStampFieldName, MetaFieldName}) {
+		log.Error("schema contains system field",
```

Comment:
better to change to Info, you can modify this in next pr

Reply:
fixed